ci.sh

```diff
@@ -37,13 +37,15 @@ python -c "import sys, struct, ssl; print('python:', sys.version); print('versio
 echo "::endgroup::"
 
 echo "::group::Install dependencies"
-python -m pip install -U pip uv -c test-requirements.txt
-python -m pip --version
+curl-harder https://astral.sh/uv/0.4.30/install.sh -o uv-install.sh
```

Comment:
If we would do this, I would suggest grepping for uv version from test-requirements.txt with `grep 'uv=='` and then cut by '==' to get version, otherwise autodeps will never be actually changing the version of `uv` used.

Reply:
Yeah I thought about that

Reply:
I've done this now